stories/link-bars/index.js

```diff
@@ -38,4 +39,8 @@ export default () => {
 		.add('... which should adjust for nested padding', ScrollArrowsNestedPaddingStory)
 		.add('... which is not a top level component', ScrollArrowsSubLevelStory)
 		.add('... which have gradient background', ScrollArrowsGradientBackground);
+
+	storiesOf('Link Bars|Carousel', module)
```

Comment:
Please move the story from Link Bars to Web App Components, and divide the story into a separate one for horizontal and vertical.